GVFS/GVFS.Service/GVFSService.Mac.cs

```diff
@@ -90,6 +88,21 @@ private void ServiceThreadMain()
             }
         }
 
+        private void AutoMountReposForUser()
+        {
+            int currentUser;
+            if (int.TryParse(this.gvfsPlatform.GetCurrentUser(), out currentUser))
+            {
+                // On Mac, there is no separate session Id. currentUser is used as sessionId
+                this.repoRegistry.AutoMountRepos(currentUser.ToString(), currentUser);
+                this.repoRegistry.TraceStatus();
+            }
+            else
+            {
+                this.tracer.RelatedError($"{nameof(this.AutoMountReposForUser)} Error: could not parse current user info from RepoRegistry.");

```

Comment:
Capture the value of `this.gvfsPlatform.GetCurrentUser()` before the call to `TryParse` so that we can log it here?